addon/mixins/axis-titles.js

```diff
@@ -126,17 +156,19 @@ const AxisTitlesMixin = Ember.Mixin.create({
    * Position of x axis title on the y axis
    * @type {Number}
    */
-  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
-    return this.get('graphicBottom') + this.get('axisPadding');
+  xAxisPositionY: Ember.computed('graphicBottom', 'labelHeightOffset',
```

Comment:
uhm, I'm not sure why we need to keep this function as it is now for consistency. Ember.computed.sum should make it look cleaner here.